openmetadata-ui/src/main/resources/ui/src/components/AddIngestion/AddIngestion.component.tsx

```diff
@@ -415,6 +417,8 @@ const AddIngestion = ({
       prevStep = activeIngestionStep - 2;
     } else if (showDBTConfig && activeIngestionStep === 4) {
       prevStep = activeIngestionStep - 2;
+    } else if (showDBTConfig && activeIngestionStep === 4) {
+      prevStep = activeIngestionStep - 2;
```

Comment:
is this duplicated?

Reply:
yup some lingering UI code from merging a previous branch. @chirag-madlani will chip in. Thanks

Reply:
thanks